modules/n1ql/pages/n1ql-language-reference/typefun.adoc

```diff
@@ -44,7 +233,51 @@ Returns one of the following strings, based on the value of expression:
 * "object"
 * "binary"
 
+== TYPE(expression)
+
+=== Description
+
+Returns one of the following strings, based on the value of expression:
+
+* "missing"
+* "null"
+* "boolean"
+* "number"
+* "string"
+* "array"
+* "object"
+* "binary"
+
+=== Examples
+====
+[source,n1ql]
+----
+SELECT TYPE(true) AS `boolean`,
+       TYPE(MISSING) AS `missing`,
+       TYPE(NULL) AS `null`,
+       TYPE(123) AS `number`,
+       TYPE("hello world") AS `string`,
+       TYPE([1, 2, 3]) AS `array`,
+       TYPE({"hello": "world"}) AS `object`;
+----
+
+[source,json]
+----
+[
+  {
+    "array": true,
+    "boolean": false,
+    "null": null,
+    "number": false,
+    "object": false,
+    "string": false
+  }
+]
+----
+====
+
 == Type-Conversion Functions
+:description: The following converts the supplied expression to a (data type).
 
 TOARRAY(expression)
```

Comment:
Do the same changes (as described above) for each of the type-conversion functions.